torch/ao/nn/quantized/reference/modules/rnn.py

```diff
@@ -294,10 +301,14 @@ def __init__(self, mode: str, input_size: int, hidden_size: int,
             for wn in self._flat_weights_names:
                 if wn.startswith("weight"):
                     weight_qparams_dict[wn] = weight_qparams
+            weight_qparams_dict["is_decompoed"] = False  # type: ignore[assignment]
```

Comment:
nit: remove?

Reply:
or, fix typo.  Not sure what the intent was.

Reply:
probably a mistake, I think I updated L300 to include this